### PR TITLE
Add lifebind module

### PR DESCRIPTION
**Pull request by trevorm4**
### Description

Show breakdown of healing that triggered lifebind heals

### Motivation

To show how people are spending their lifebinds

### Testing
image

### CODE REVIEW

File: src/analysis/retail/evoker/preservation/constants.ts [hunk 1]
```diff
@@ -18,6 +18,26 @@ export const SPELL_COLORS = {
...
 };
 
+export const CORE_HEALING_SPELLS = [
+  SPELLS.EMERALD_BLOSSOM.id,
+  SPELLS.EMERALD_BLOSSOM_ECHO.id,
+  TALENTS_EVOKER.REVERSION_TALENT.id,
+  SPELLS.REVERSION_ECHO.id,
+  SPELLS.SPIRITBLOOM.id,
+  SPELLS.SPIRITBLOOM_SPLIT.id,
+  SPELLS.DREAM_BREATH.id,
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/constants.ts:30]: Dream Breath Echo is missing

File: src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      spellID,
+      (this.healingBySpell.get(spellID) ?? 0) + event.amount + (event.absorbed || 0),
+    );
+  }
+
+  healingForSpell(spellId: number): number {
+    return this.healingBySpell.get(spellId) ?? 0;
+  }
+
+  renderDonutChart() {
```
Comment (vohrr) [inline src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx:36]: Not sure if intended or not but this chart doesn't have all of the spells from your core healing spells obj.
Missing: 
Rewind, Dream Flight, Cycle of Life, Fluttering Seedlings, Echo,

**Review by vohrr — CHANGES_REQUESTED** [review]
Couple things, mostly good to go

File: src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx (hunk 2 above)
Comment (trevorm4) [reply src/analysis/retail/evoker/preservation/modules/talents/Lifebind.tsx:36]: Yeah you're right, I think I started adding them and didn't finish it. Let me add the rest.

**Review by vohrr — APPROVED** [review]

**Pull request merged**